docs/database_example.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>Please import only the necessary stuff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
We decided to just do * imports for this, as it should just be a quick example. We felt that importing everything here looks overwhelming for users if they just want to step through this thing quickly.

Reply:
Not sure why this would complicate things. I think it is still useful for any user to get familiar ASAP with what is imported from where...but if this is sth you have already discussed there, I am fine with it.